partner_password_reset/models/res_partner.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.multi
+    def action_reset_user_password(self):
+        for rec in self:
+            rec.user_ids.action_reset_password()
```

Comment:
Yes it is, please see:

https://github.com/odoo/odoo/blob/9.0/openerp/addons/base/res/res_partner.py#L246

`user_id` is definitely not the droid I'm looking for:
https://github.com/odoo/odoo/blob/9.0/openerp/addons/base/res/res_partner.py#L198

😆